src/pages/home/report/ReactionList/HeaderReactionList.tsx

```diff
@@ -0,0 +1,46 @@
+import {View} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as EmojiUtils from '@libs/EmojiUtils';
+import type ReactionListProps from './types';
+
+type HeaderReactionListProps = ReactionListProps & {
+    /** Returns true if the current account has reacted to the report action (with the given skin tone). */
+    hasUserReacted: boolean;
```

Comment:
Please double check all props that you migrated, I think this was optional before

Reply:
Same as https://github.com/Expensify/App/pull/34549#discussion_r1465126372
I checked the types and it seems that this specific one was required.

Reply:
Same, make it optional and use a default value 👍

Reply:
Done.
I also remove `onClose` prop from this component since it wasn't getting used at all.